lib/matplotlib/figure.py

```diff
@@ -1220,12 +1220,16 @@ def colorbar(
                 fig.colorbar(cm.ScalarMappable(norm=norm, cmap=cmap), ax=ax)
 
         cax : `~matplotlib.axes.Axes`, optional
-            Axes into which the colorbar will be drawn.
+            Axes into which the colorbar will be drawn.  If `None`, then space
+            will be stolen from the Axes(s) specified in *ax*
 
         ax : `~.axes.Axes` or iterable or `numpy.ndarray` of Axes, optional
-            One or more parent axes from which space for a new colorbar axes
-            will be stolen, if *cax* is None.  This has no effect if *cax* is
-            set.
+            If `None`, defaults to ``mappable.axes``.
```

Comment:
mappable.axes could be clarified a bit.  
```suggestion
            If `None`, defaults to the axes that contains the mappable used to create the colorbar (``mappable.axes``).
```

Reply:
I agree. I think the text explanation is enough and we do not need to mention the technical ``mappable.axes``, which likely almost nobody knows about.